lib/xboxrt/libc_extensions/wchar_ext_.h

```diff
@@ -0,0 +1,2 @@
+// Windows-specific extension required by libc++
+int _snwprintf(wchar_t *buffer, size_t count, const wchar_t *format, ...);
```

Comment:
Wouldn't this need `#ifdef __cplusplus` ?

Reply:
Yes it does, fixed.